client/reader/user-stream/style.scss

```diff
@@ -11,4 +11,17 @@
 			border-block-end: none;
 		}
 	}
+
+	.user-profile__lists-body {
+		max-width: 768px;
+		margin: 0 auto;
+
+		.user-profile__lists .reader-post-card__title {
+			margin-top: 0;
+
+			.reader-post-card__title-link {
+				font-size: 1.25rem;
+			}
+		}
+	}
```

Comment:
Just realized I can probably clean up these class names a bit. Will do that tomorrow.